src/taskrunner.py

```diff
@@ -0,0 +1,241 @@
+import sys, os, traceback
+import cfn, lifecycle, masterless, vault, aws, metrics, tasks, master, askmaster, buildvars, project, deploy
+from decorators import echo_output
+
+@echo_output
+def ping():
+    return "pong"
+
+@echo_output
+def echo(msg, *args, **kwargs):
+    if args or kwargs:
+        return "received: %s with args: %s and kwargs: %s" % (msg, args, kwargs)
+    return "received: %s" % (msg,)
+
+UNQUALIFIED_TASK_LIST = [
```

Comment:
some documentation of these long lists will help placing a new command in the right place